src/status_im/contexts/wallet/send/select_address/tabs/view.cljs

```diff
@@ -27,11 +17,17 @@
         :description     (i18n/label :t/here-is-a-cat-in-a-box-instead)
         :image           (resources/get-themed-image :cat-in-box theme)
         :container-style style/empty-container-style}]
-      [gesture/flat-list
```

Comment:
Since the parent already uses a scroll view and we had to remove the flat list as it was throwing the error:

```
VirtualizedLists should never be nested inside plain ScrollViews with the same orientation because it can break windowing and other functionality - use another VirtualizedList-backed container instead.
```

Now, it uses the plain old for loop to render the account items. Generally, account items

Reply:
Two things:

1. If the parent is always a `scroll-view`, we can replace the `rn/view` wrapping this `(doall ...)`  by `[:<>]` and provide its styles to the `scroll-view` (inside `:content-container-style` I think)

2. If we aren't dyncamically `deref`-ing an ratom inside this lazy seq (created by `for`), we can omit the `doall` and keep it lazy, otherwhise, if we need to realize this seq, then we could use `into` so that we no longer need the meta `:key` given, something like:
```clojure
(into [:<>] ;; or rn/view
      (map (fn [{:keys [color address] :as account}]
             [quo/account-item {,,,}]))
      other-accounts)
```